dcompiler.dd

```diff
@@ -37,7 +37,7 @@ $(H2 $(LNAME2 requirements, Requirements and Downloads))
     $(LINUX
     $(LI 32 bit x86 and 64 bit x86-64 Linux operating system)
 
-    $(LI Gnu C compiler (gcc))
+    $(LI Gnu C compiler (gcc) or LLVM C compiler (clang))
```

Comment:
Can `clang` be used for `/bin/cc` for linking, then?

Reply:
Yes, works fine on my machine:

![image](https://user-images.githubusercontent.com/4370550/35928183-cf49e08e-0c2c-11e8-8ae5-bafb36a65c08.png)

(though it would be nice to have a switch to change the linker)

Reply:
Try it yourself with:

```
echo 'void main(){import std.stdio; "hello".writeln;}' | CC=/usr/bin/clang dmd -v -run
```